include/mgba/core/core.h

```diff
@@ -160,6 +160,9 @@ struct mCore {
 	void (*startVideoLog)(struct mCore*, struct mVideoLogContext*);
 	void (*endVideoLog)(struct mCore*);
 #endif
+
+	size_t (*hwExtensionsSerialize)(struct mCore*, void** sram);
+	bool (*hwExtensionsDeserialize)(struct mCore*, const void* sram, size_t size);
```

Comment:
Would probably make more sense to have a "generic" extdata serialize that you specify via passing in an extdata flag.

Reply:
Fixed.